src/ui/playblast_ui.py

```diff
@@ -14,7 +14,8 @@
 ICON = join(icon_path, "ui", "folder.png")
 
 def get_maya_window():
-    """Get pointed to Maya's main window to use as parent
+    """
+    Get pointed to Maya's main window to use as parent
```

Comment:
```suggestion
    Get pointer to Maya's main window to use as parent
```

Pointer, no "pointed"